### PR TITLE
Small patches to upload* in core

### PR DESCRIPTION
**Pull request by janiversen**
### Describe the pull request:

- [x ] Bug fix
- [ ] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:

During the qml integration, the extended testing and the resulting debugging a couple of issues
was found, some more serious than others.

The commits are small, to include only 1 problem.

This must be merged BEFORE #2435 is merged

### CODE REVIEW

File: mobile-widgets/qml/Export.qml [hunk 1]
```diff
@@ -75,7 +75,6 @@ Kirigami.ScrollablePage {
 		onApply: {
 			if (selectedExport === ExportType.EX_DIVELOGS_DE) {
 				if (fieldUserID.text !== PrefCloudStorage.divelogde_user) {
```
Comment (dirkhh) [inline mobile-widgets/qml/Export.qml:77]: interestingly, our coding style would suggest to remove the curly braces here...

Comment (dirkhh) [reply mobile-widgets/qml/Export.qml:77]: not asking you to change this, just rambling

**Review by dirkhh — APPROVED** [review]
thanks for this PR. I love the granularity of the commits.

File: mobile-widgets/qml/Export.qml (hunk 1 above)
Comment (janiversen) [reply mobile-widgets/qml/Export.qml:77]: I will bear that in mind, to be honest I tend to use {} too often (I have too often seen If if else go wrong :-)

**Pull request merged**